### PR TITLE
Upgraded http4s stable to 0.23.14 and milestone to 1.0.0-M35

### PR DESCRIPTION
**Pull request by ovo-vholovko**
Just version upgrades.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -138,7 +139,7 @@ lazy val natchezHttp4s = projectMatrix
       name := "natchez-extras-http4s-stable",
       libraryDependencies ++= Seq(
         "org.http4s" %% "http4s-dsl" % http4sStableVersion,
-        "org.http4s" %% "http4s-client" % http4sStableVersion
+        "org.http4s" %% "http4s-client" % blazeStableVersion
```
Comment (voidcontext) [inline build.sbt:142]: See above.

File: build.sbt [hunk 2]
```diff
@@ -89,7 +90,7 @@ lazy val natchezDatadog = projectMatrix
       libraryDependencies ++= Seq(
         "org.http4s" %% "http4s-dsl" % http4sStableVersion,
         "org.http4s" %% "http4s-circe" % http4sStableVersion,
-        "org.http4s" %% "http4s-client" % http4sStableVersion
+        "org.http4s" %% "http4s-client" % blazeStableVersion
```
Comment (voidcontext) [inline build.sbt:93]: According to the README here: https://github.com/http4s/blaze#standalone-modules

Only `http4s-blaze-*` packages have been moved out from the main repo. As far as I understands the `http4s-client` package is implementation agnostic, and follows the main version.

**Review by voidcontext — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 1 above)
Comment (ovo-vholovko) [reply build.sbt]: yep, reverted to the original version, thanks!

File: build.sbt (hunk 2 above)
Comment (ovo-vholovko) [reply build.sbt]: returned to the original version, thanks!

**Review by voidcontext — APPROVED** [review]

**Pull request merged**